tsconfig.json

```diff
@@ -1,14 +1,14 @@
 {
-  "include": ["src", "types"],
+  "include": ["src", "test", "stories"],
   "compilerOptions": {
     "module": "ES6",
     "lib": ["es2020", "dom"],
-    "target": "ES6",
+    "target": "es2016",
```

Comment:
why is 2016?

Reply:
Good question. I started to have issues with support for `async/await` and so thought I might as well bump to the version which supports it so I don't have to bring in polyfills/babel transforms. I figure es2016 is still rather old when we typically say things like we support the last two versions of browsers.